src/components/bcros/filing/common/HeaderActions.vue

```diff
@@ -1,5 +1,57 @@
 <template>
   <div class="flex flex-row items-center">
+    <BcrosDialog
+      name="filingModal"
+      :display="showFilingModal"
+      :options="filingDialogOptions"
+      @close="setShowFilingModal(false)"
+    >
+      <template #content>
+        <UAlert
+          v-if="filingError !== ''"
```

Comment:
NB: you can usually just use `v-if="filingError"` this has added benefit it also fails if filingError is undefined or null 

here, if you pass number, or object, or undefined or null, etc... to the filingError, it will show the alert